### PR TITLE
RGRIDT-832: Fixing export excel with styles

### PR DESCRIPTION
**Pull request by glundgren93**
This PR fixes conditional format not being applied on excel export.

### What was happening
* Conditional format was not being applied on excel export.

### What was done
* Adjusted exportToExcel method in order to apply conditional format on all cells

### Checklist
* [x] tested locally
* [x] documented the code
* [x] clean all warnings and errors of eslint
* [x] requires changes in OutSystems (if so, provide a module with changes)
* [ ] requires new sample page in OutSystems (if so, provide a module with changes)

### CODE REVIEW

File: code/src/WijmoProvider/Features/Export.ts [hunk 1]
```diff
@@ -82,24 +82,22 @@ namespace WijmoProvider.Feature {
...
-                params
-            );
-            //name the sheet
-            book.sheets[0].name = 'DataGrid Data';
-            // save the book
-            book.save('DataGridReactive.xlsx');
-
-            this._reApplyPagination();
+            // include timeout in order to apply conditional format
+            setTimeout(() => {
```
Comment (tmbp95) [inline code/src/WijmoProvider/Features/Export.ts:86]: We were doing this exact same setTimeout for the trad web, right?

Comment (glundgren93) [reply code/src/WijmoProvider/Features/Export.ts:86]: yea

**Review by tmbp95 — APPROVED** [review]
I wish we were able to avoid the setTimout :(

File: code/src/WijmoProvider/Features/Export.ts (hunk 1 above)
Comment (bmartinho) [reply code/src/WijmoProvider/Features/Export.ts:86]: Let's keep this also in Reactive

**Review by bmartinho — APPROVED** [review]
Ready to go

**Pull request merged**